funding/manager.go

```diff
@@ -705,6 +705,7 @@ func (m *Manager) BatchChannelSetup(
 					matchedOrderBid.SelfChanBalance,
 				),
 				CommitmentType: commitmentType,
+				Private:        matchedOrderBid.PrivateChannel,
```

Comment:
I think even if we don't spill things over into the ask side of things, we also want to add a channel acceptor check here to ensure the channel is properly formed. This becomes more important when someone wants to buy a: unadvertised zero conf channel.

Reply:
Channel acceptor added